test/sparse/sparse.jl

```diff
@@ -47,12 +47,18 @@ end
 @testset "concatenation tests" begin
     @testset "horizontal concatenation" begin
         @test all([se33 se33] == sparse([1, 2, 3, 1, 2, 3], [1, 2, 3, 4, 5, 6], ones(6)))
+
+        sp33 = speye(3, 3)
```

Comment:
This `sp33` definition seems unnecessary given its absence in the immediately following test, and likewise two lines below? Or am I missing something? (Also, why the additional empty lines?)